scripts/fire-reboot-normal

```diff
@@ -2,12 +2,14 @@
 logger "FIREWALLA REBOOT NORMAL"
 #sudo sh -c 'echo V > /dev/watchdog'
 #sudo sh -c 'echo V > /dev/watchdog'
-sudo pkill bitbridge
-sudo pkill fire-ping
-sudo pkill sleep
+sudo service firemain stop
```

Comment:
what about the other services? fireapi, firemon, firekick?

Reply:
fireapi can't be stopped, otherwise rest of the process won't execute, and never reboot.  Rest services don't need stop.  We stopping fireman is to recover network